examples/android.rs

```diff
@@ -0,0 +1,24 @@
+use app_dirs2::*;
+use log::info;
+
+const NDK_APP_INFO: AppInfo = AppInfo {
+    name: "unnecessary",
```

Comment:
I'm also creating this example to see if anyone else feels like adding a `NO_NAME` similar to `NO_AUTHOR`. On Android all these directories already contain the app/package name inside, and have thus little use for an extra name in the path, as shown in the PR message.